src/test/scala/ilc/feature/let/BetaReductionSuite.scala

```diff
@@ -83,8 +83,8 @@ extends FlatSpec
   val duplicating = lambda(Var("f", NatType =>: NatType), Var("expensive", NatType)) {
     case Seq(arg, expensive) => PlusNat ! (arg ! expensive) ! (arg ! expensive)
   } ! u ! (PlusNat ! 42 ! 84)
-  pretty(duplicating)
-  pretty(normalize(duplicating))
+  info("\n" + pretty(duplicating))
+  info("\n" + pretty(normalize(duplicating)))
```

Comment:
[1]: https://github.com/inc-lc/ilc-scala/blob/topic/pretty-by-kiama/src/test/scala/ilc/feature/functions/PrettySuite.scala#L16

The method is in [`feature.functions.PrettySuite`][1]. The call to `pretty` here looks provisional, and there's some commented-out code block below. I assumed this test is in the middle of development by @Blaisorblade.